configs/setups/awicm3/awicm3.yaml

```diff
@@ -1,196 +1,657 @@
+#                type: shell
+#
+#postprocess:
+#        postprocess_shell:
+#                method: "${general.esm_namelist_dir}/../configs/components/oifs/oifs-43r3-postprocess.sh ${work_dir} ECE3 ${start_date!syear!smonth!sday} ${end_date!syear!smonth!sday}"
+#                type: shell
 
 
 #########################################################################################
 ########### necessary changes to submodels compared to standalone setups ################
 #########################################################################################
 
-echam:
-        restart_firstlast: "first"
-        namelist_changes:
-                namelist.echam:
-                        runctl:
-                                lcouple: .true.
-        adj_input_dir: "${fesom2.mesh_dir}/tarfiles${echam.resolution}/input/echam6"
-        model_dir: ${general.model_dir}/echam-${echam.version}
-        setup_dir: ${general.model_dir}
-        ocean_resolution: "${fesom2.resolution}"
-        remove_forcing_files:
-                - sst
-                - sic
-        version: "6.3.04p1"
+oifs:
+        version: "43r3"
 
-        choose_general.resolution:
-                T63_CORE2:
-                        nproca: 24
-                        nprocb: 18
+        execution_command: ${executable} -v ecmwf -e ${oifs.input_expid}
 
+        pool_dir: ${general.pool_dir}
+        model_dir: ${general.model_dir}/oifs-${version}
+        setup_dir: ${general.model_dir}
+        restart_firstlast: "last"
+        restart_rate: 12
+        ocean_resolution: "${fesom.resolution}"
+
+        mip: "cmip6"
+        scenario: "historical"
+
+        prepifs_expid: h6mv
+        input_expid: awi3   # This cannot be longer than 4 characters
+        input_dir: ${pool_dir}/oifs-${version}
+        rtables_dir: /work/ollie/jstreffi/input/oifs-43r3/rtables/
+        ##vtables_dir: ${input_dir}/../vtables/
+        ##clim_dir: ${input_dir}/../
+        ##icmcl_dir: ${input_dir}
+        ##icmcl_file: ICMCLgu5aINIT
+        forcing_dir: ${input_dir}/${version}/ifsdata/
+        ifsdata_dir: ${input_dir}/${version}/ifsdata/
+        namelist_dir: ${general.esm_namelist_dir}/oifs/${version}/awicm3/202011
+        cmip6_data_dir_nml: ${input_dir}/cmip6-data
+        cmip5_data_dir_nml: ${input_dir}/cmip5-data
+
+
+        # nprocb must be 1
+        # nproca must be not a prime number for OpenIFS parallelisation to work
+        # (This is just one of the many mysteries of the IFS...)
+        #
+        # We want to use 1 CPU for runoff mapper
+        # 287 fulfills this requirement and 288 divides well with 36 and 48
+        # (287 = 41 * 7)
+        # so that the configuration fills up the compute nodes
+        # 575 would also be ok (575 = 23 * 25)
+        nproca: 287
+        nprocb: 1
+
+        # By default we turn on ECWAM wave model (simulation FR10) but it should be
+        # false for DRH6
+        wam: true
```

Comment:
Since you are running something like the DRH6 exp for now, and that one had no wam, we probably need this set for false for now. It can be changed with a small patch afterwards

Reply:
Sounds good to me. I want to use it, and I think it works, but I always struggled with the output. 
Let's make "false" the default. Some testing is needed to justify "true".

Reply:
@joakimkjellsson , this line is defined only in `awicm3.yaml`, in the oifs section, meaning changing this line would not change the default you set in `oifs.yaml`. Do you want me to switch it to false also in the `oifs.yaml`?

Reply:
Do it! ;-)